diagnostic.md

```diff
@@ -7,27 +7,32 @@ Write your answers inside this file, where it's indicated by the comments.
     repository in your answer.
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    git clone git@github.com:byau/git-basics-diagnostic.git
     ```
 
 2.  What do you do after cloning a repository, but before starting work?
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    cd git-basics-diagnostic
```

Comment:
I suppose that is technically true, but it's not what we were looking for. What should you do before starting to make your own changes?